src/sentry/models/orgauthtoken.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils import timezone
+from django.utils.encoding import force_text
+
+from sentry.db.models import (
+    ArrayField,
+    BaseManager,
+    FlexibleForeignKey,
+    Model,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.models.orgauthtokenproject import OrgAuthTokenProject
+
+
+@control_silo_only_model
+class OrgAuthToken(Model):
+    __include_in_export__ = True
+
+    organization = FlexibleForeignKey("sentry.Organization", null=False)
+    projects = models.ManyToManyField(
+        "sentry.Project", related_name="projects", through=OrgAuthTokenProject
+    )
+    # max JWT token length is 2048, +7 for our custom prefix (sntrys_)
+    token = models.CharField(max_length=2055, unique=True, null=False)
```

Comment:
Not 100% sure what I should make the max length here. on the RFC there is a comment that we should store this in hashed form, if we do this I guess we need a more generic length here...?

Reply:
@mitsuhiko why would we need to store it hashed?

Reply:
see: https://github.com/getsentry/rfcs/pull/91#discussion_r1218186407

Reply:
Great thanks! I supposed it was for security reasons then it makes sense.

Here we should make the length, the length of the hash. I don't expect we will need to store the token in any other format besides the hashed one.

Reply:
Are you planning on storing the encoded JWT? or only the important claims within the token?

Reply:
I updated this to be explicit that we store `token_hashed` in the DB.